can/interfaces/ics_neovi/neovi_bus.py

```diff
@@ -158,15 +158,23 @@ def __init__(self, channel, can_filters=None, **kwargs):
         with open_lock:
             ics.open_device(self.dev)
 
-        if "bitrate" in kwargs:
-            for channel in self.channels:
-                ics.set_bit_rate(self.dev, kwargs.get("bitrate"), channel)
-
-        fd = kwargs.get("fd", False)
-        if fd:
-            if "data_bitrate" in kwargs:
+        try:
+            if "bitrate" in kwargs:
                 for channel in self.channels:
-                    ics.set_fd_bit_rate(self.dev, kwargs.get("data_bitrate"), channel)
+                    ics.set_bit_rate(self.dev, kwargs.get("bitrate"), channel)
+
+            fd = kwargs.get("fd", False)
+            if fd:
+                if "data_bitrate" in kwargs:
+                    for channel in self.channels:
+                        ics.set_fd_bit_rate(
+                            self.dev, kwargs.get("data_bitrate"), channel
+                        )
+        except ics.RuntimeError as re:
+            logger.error(re)
+            err = ICSApiError(*ics.get_last_api_error(self.dev))
+            self.shutdown()
```

Comment:
Just a thought: can this throw an exception again? Would we also want to catch that?

Reply:
Yes, `shutdown` (which call `ics.close_device` can raise a `ics.RuntimeError`. However, in this case we do not want to call `ics.get_last_api_error`.

Will fix

Reply:
Cool!